### PR TITLE
Cleans_stale_notice

### PR DESCRIPTION
**Pull request by pschoffer**
## Description

Adds removing stale dismissed identifiers in every ~ 1 in 100 admin requests.

Also sneaked in the fix for adding multiple dismissibility keys.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [x] This change has relevant unit tests (if applicable).
- [x] This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Define few (at least 3) notices that are dismissible
2. Dismiss two of them
3 On refresh only see one remaining

### CODE REVIEW

File: admin-notice/class-admin-notice-controller.php [hunk 1]
```diff
@@ -41,8 +43,35 @@ public function dismiss_vip_notice() {
 		check_ajax_referer( self::DISMISS_NONCE_ACTION );
 
 		if ( isset( $_POST[ self::DISMISS_IDENTIFIER_KEY ] ) ) {
-			update_user_meta( get_current_user_id(), self::DISMISS_USER_META, sanitize_text_field( $_POST[ self::DISMISS_IDENTIFIER_KEY ] ) );
+			add_user_meta( get_current_user_id(), self::DISMISS_USER_META, sanitize_text_field( $_POST[ self::DISMISS_IDENTIFIER_KEY ] ) );
```
Comment (pschoffer) [inline admin-notice/class-admin-notice-controller.php:46]: Fix dismissing multiple notices bug.

**Review by netsuso — APPROVED** [review]
Everything looks good! I've tested both the fix for multiple dismisses and the cleanup of stale dismissed meta, and it worked as expected :)

**Comment by netsuso** [comment]
stacks-r1287

File: admin-notice/class-admin-notice-controller.php (hunk 1 above)
Comment (GaryJones) [reply admin-notice/class-admin-notice-controller.php:46]: Won't this only persist the first dismissed notice, and not any more?

**Pull request merged**